qif.py

```diff
@@ -77,7 +77,11 @@ def parse_qif(infile):
             curItem = QifItem()
             curItem.account = account
         elif firstchar == 'D':
-            year, month, day = map(int, data.split('/'))
+            month, day, year = map(int, data.split('/'))
+            if month > 1000:
+                year, month, day = map(int, data.split('/'))
+            elif month > 12:
```

Comment:
that won't reliably detect month/day vs day/month

Reply:
Yes, I agree. Do you have a better approach? QIF specs say to use mm/dd/yyyy (as far as I understand), so the correct option would actually be to remove the dd/mm/yyyy possibility (or even yyyy/mm/dd). But that would certainly break the usage for everyone who has used the qif script so far.

Reply:
the script used the order "year, month, day" before, i.e. I would only remove the "elif month > 12" part as it's not reliable (but we can assume month > 1000 to detect a year is)

Reply:
Hi hjacobs, only saw your comment just now... I am fine with your approach. Do I have to resubmit my changes or can you do the changes directly?